restlight-jaxrs-provider/src/main/java/io/esastack/restlight/jaxrs/adapter/MessageBodyReaderAdapter.java

```diff
@@ -39,22 +39,22 @@ public MessageBodyReaderAdapter(Providers providers) {
 
     @SuppressWarnings("unchecked")
     @Override
-    public HandledValue<Object> readFrom(RequestEntity entity, RequestContext context) throws Exception {
+    public Result<?, Void> readFrom(RequestEntity entity, RequestContext context) throws Exception {
         Class<?> type;
         if ((type = entity.type()) == null) {
-            return HandledValue.failed();
+            return Result.err();
         }
         MediaType mediaType = MediaTypeUtils.convert(entity.mediaType());
         MessageBodyReader<T> reader = (MessageBodyReader<T>) providers.getMessageBodyReader(type,
                 entity.genericType(),
                 entity.annotations(),
                 mediaType);
         if (reader == null) {
-            return HandledValue.failed();
+            return Result.err();
```

Comment:
Err is a more general expression. In fact, if we really want to use a more accurate expression, failed is not very appropriate too. We should express it with unsupported. But for reuse, we reuse result.err(), instead of starting a new stove.in this case, there is no need to be too tangled about whether the expression is completely accurate, and the meaning is roughly the same.